Code/Editor/Util/MemoryBlock.cpp

```diff
@@ -169,13 +169,12 @@ void CMemoryBlock::Uncompress(CMemoryBlock& toBlock) const
     assert(this != &toBlock);
     toBlock.Allocate(m_uncompressedSize);
     toBlock.m_uncompressedSize = 0;
-    unsigned long destSize = m_uncompressedSize;
 #if !defined(NDEBUG)
-    int result =
-#endif
-        uncompress((unsigned char*)toBlock.GetBuffer(), &destSize, (unsigned char*)GetBuffer(), GetSize());
+    unsigned long destSize = m_uncompressedSize;
+    int result = uncompress((unsigned char*)toBlock.GetBuffer(), &destSize, (unsigned char*)GetBuffer(), GetSize());
```

Comment:
Is this right? The original logic looks like (as bad as it is) will still in debug uncompress, with the only difference being result is set and asserted on.